src/classic.c

```diff
@@ -10,5 +10,6 @@ bool is_running_on_classic_distribution()
 	    || access("/var/lib/dpkg/status", F_OK) == 0
 	    || access("/var/lib/pacman", F_OK) == 0
 	    || access("/var/lib/portage", F_OK) == 0
-	    || access("/var/lib/rpm", F_OK) == 0;
+	    || access("/var/lib/rpm", F_OK) == 0
+	    || access("/sbin/procd", F_OK) == 0;
```

Comment:
OpenWrt has a build time configuration option: "Remove ipkg/opkg status data files in final images" which can lead again to fake classic detection